mcs/build/platforms/win32.make

```diff
@@ -10,7 +10,7 @@ PLATFORM_TEST_HARNESS_EXCLUDES = NotOnWindows,
 
 EXTERNAL_MCS = mcs
 EXTERNAL_MBAS = vbc.exe
-EXTERNAL_RUNTIME =
+EXTERNAL_RUNTIME = mono.bat
```

Comment:
We don't have mono.bat anymore, just normal mono.exe. You need to replace this with just `mono`.